lib/lob/resources/resource_base.rb

```diff
@@ -12,41 +12,71 @@ def initialize(config)
         @config = config
       end
 
-      def list(options={})
-        submit :get, endpoint_url, options
+      def list(query={})
+        request = {
+          method: :get,
+          url: endpoint_url,
+          query: query
+        }
+
+        submit request
       end
 
       def find(resource_id)
-        submit :get, resource_url(resource_id)
+        request = {
+          method: :get,
+          url: resource_url(resource_id)
+        }
+
+        submit request
       end
 
-      def create(options={}, headers={})
-        submit :post, endpoint_url, options, headers
+      def create(body={}, headers={}, query={})
```

Comment:
I don't understand this change. Why does `create` need to change for the base?

Reply:
The create base previously took a POST body (`options`) and `headers` (used for idempotency keys). We also wanted to support query params, so the base `create` method had to be updated.